src/Models/VerificationCode.php

```diff
@@ -20,12 +22,22 @@ class VerificationCode extends Model
         'action_name',
         'expires_at',
         'verifies_until',
-        'used_at'
+        'used_at',
+        'last_touched_at',
+        'host_ip'
     ];
 
     public function scopeAllFor(Builder $query, Verifiable $verifiable)
     {
         return $query->where('verifiable_type', $verifiable->getMorphClass())
-                     ->where('verifiable_id', $verifiable->getKey());
+                     ->where('verifiable_id', $verifiable->getKey())
+                     ->where('host_ip', Container::getInstance()->make('request')->ip());
```

Comment:
Je to Container::getInstance()->make('request')->ip() potrebné? request()->ip() tu nefunguje?

Reply:
@palypster - No praveze takto nejako sme s Kikou uvazovali, aby sme chranili akciu daneho usera pre jeho konkretne zariadenie, cize preto je tam aj IP. A to prave preto, aby sa nestalo, ze niekto ziska pristup k tvojmu uctu a z ineho zariadenia vykona akciu, na ktoru si sa verifikoval na inom zariadeni. Z hladiska bezpecnosti je toto podla mna dobry krok.

Reply:
Nerozumieme si, jasne ze to musime checkovat. Otazka bola ci na urovni DB alebo druha moznost bola, ze si querines vsetky a NASLEDNE v PHP spravis ten check. Akoze toto je rychlejsie of course, nenapada mi aktualne dovod, preco by to malo byt v PHP, takze ok, nechajme takto.